src/Microsoft.Win32.Registry/ref/Microsoft.Win32.Registry.cs

```diff
@@ -105,24 +110,31 @@ public enum RegistryView
         Registry32 = 512,
         Registry64 = 256,
     }
-    public enum RegistryKeyPermissionCheck
-    {
-        Default = 0,
-        ReadSubTree = 1,
-        ReadWriteSubTree = 2,
-    }
 }
 namespace Microsoft.Win32.SafeHandles
 {
     public sealed partial class SafeRegistryHandle : Microsoft.Win32.SafeHandles.SafeHandleZeroOrMinusOneIsInvalid
     {
-        public SafeRegistryHandle(System.IntPtr preexistingHandle, bool ownsHandle) : base(default(bool)) { }
-        public override bool IsInvalid { get { throw null; } }
```

Comment:
`SafeRegistryHandle` doesn't override this `IsInvalid` property.
https://github.com/dotnet/corefx/blob/master/src/Common/src/CoreLib/Microsoft/Win32/SafeHandles/SafeRegistryHandle.cshttps://github.com/dotnet/corefx/blob/master/src/Common/src/CoreLib/Microsoft/Win32/SafeHandles/SafeRegistryHandle.Windows.cs
https://github.com/dotnet/corefx/blob/master/src/Microsoft.Win32.Registry/src/Microsoft/Win32/SafeHandles/SafeRegistryHandle.FileSystem.cs